src/app/api_services/vtmapi.service.spec.ts

```diff
@@ -98,4 +121,29 @@ describe('VtmapiService', () => {
       expect(detailsData?.faction).toBe(testData.faction);
     });
   });
+
+  it('can test for network error', () => {
+    const emsg = 'simulated network error';
+
+    service.getRaces().subscribe(
+      data => {
+        expect(data).toBeUndefined();
+      },
+      // Shouldn't be used since if there is a network error an empty result is passed from the dndapiservice handleError method
+      // so it gets handled by the about case with the data variable
```

Comment:
Would you be able to reword this to make it a bit clearer. Does the error just get handled in the dndapiservice?